app/package.json

```diff
@@ -30,7 +30,7 @@
     "dugite": "1.91.3",
     "electron-window-state": "^5.0.3",
     "event-kit": "^2.0.0",
-    "file-uri-to-path": "0.0.2",
+    "file-uri-to-path": "^2.0.0",
```

Comment:
It's hard to tell based on [the diff](https://github.com/TooTallNate/file-uri-to-path/compare/0.0.2...2.0.0) between the two versions (since the codebase got converted to TS), but based on [the diff between v0.0.2 and v1.0.0](https://github.com/TooTallNate/file-uri-to-path/compare/0.0.2...1.0.0), the changelog (that mentions that v2.0.0 only converted the codebase to TS), and [the list of commits](https://github.com/TooTallNate/file-uri-to-path/compare/1.0.0...2.0.0) between v1.0.0 and v2.0.0, I'm quite certain that the risk of this update is very low.